src/basic_recipes/title.jl

```diff
@@ -1,3 +1,10 @@
+export addtitle
+
+"""
+    title(titletext, plot)
+
+Create a `title` for `plot`. Note that the title is not automatically added to `plot`. Instead [`hbox`](@ref) can be used for this. Alternatively the [`addtitle`](@ref) convenience does create a title and add it to the plot.
+"""
 @recipe(Title, titletext, plot) do scene
```

Comment:
Would bring this more in line with `legend`'s API to have the scene argument first.

The rest looks good!